Basic API for Next

sardinger opened the pull request:
Error in [id].ts. I know it has something to do with types. Tried casting it as a string creating a check at the top like we did with the SpotifyFormFields but couldn't get it to work. Also added a lot of comments trying to explain what's happening in the code

sardinger committed:
Basic API for Next

One error

5Minty commented:
Also the [id].js isn't working properly. According  to the site this is what should be happening:
To retrieve a user by id when the page loads, we can use the React useEffect hook with the id as a dependency. The browser-supported Fetch API is used to make a GET request to /api/users/[id]. Async/await is used to wait for this asynchronous request to complete. Once it is completed, the user's name is saved to the component state using setName, and displayed within the input text box.

When the form is submitted, a POST request is made to the /api/users API route that we created earlier. To keep things simple, the response of the request is then logged to the console.

sardinger committed:
ahfahdf;

sardinger committed:
aldsflafh; 2

wdixon-bzs committed:
Lots of changes, comments, and assigned 2 tasks

sardinger committed:
Part 2 completed, auth code added

Finished part 2 of your mock API task, added Spotify's oauth code but need to fix it to work with mine
